Assets/BossRoom/Scripts/Client/Game/Action/PickUpActionFX.cs

```diff
@@ -0,0 +1,53 @@
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Visual
+{
+    /// <summary>
+    /// Complimentary FX class for the Pickup Action class. Plays a "Pickup" or "Drop" animation.
+    /// </summary>
+    public class PickUpActionFX : ActionFX
```

Comment:
any reason why we decided to still go with the action pattern instead of the 2 class pattern we discussed a few weeks ago?

Reply:
My thinking was that having two different Action systems would not be too easy to follow along through code. The two class pattern is a viable route, but there's other systems at play that would need to be accounted for (input, cross-play with other actions such as targeting, visuals, etc).